applications/serial_lte_modem/sysbuild/mcuboot/boards/thingy91x_nrf9151.overlay

```diff
@@ -0,0 +1,4 @@
+&uart0 {
+	status = "okay";
+	current-speed = < 1000000 >;
```

Comment:
Is there a reason for not having this in https://github.com/nrfconnect/sdk-nrf/blob/main/applications/serial_lte_modem/boards/thingy91x_nrf9151_ns.overlay?

Reply:
@maxd-nordic